swap_meet/vendor.py

```diff
@@ -0,0 +1,95 @@
+class Vendor:
+    
+    def __init__(self, inventory = None):
+        
+        #self.inventory = inventory
+        if inventory == None:
+            self.inventory = []
+        else:
+            self.inventory = inventory
+    
+    # method, adds item to self.inventory[]
+    def add(self,item):
+        if item not in self.inventory:
+            self.inventory.append(item)
+        return item 
+    
+    # returns and removes item if it is found in the inventory list
+    def remove(self,item):
+        if item in self.inventory:
+            self.inventory.remove(item)
+            return item
+        return False
+    
+    # .category is referencing category in Item class. This method returns list of items that match type_of_stuff category
+    def get_by_category(self, type_of_stuff):
+        categorlist = []
+        for item in self.inventory:
+            if item.category == type_of_stuff:
+                categorlist.append(item)
+        return categorlist
+
+   
+    def swap_items(self, swapping_vendor, my_item, their_item):
+        
+        if my_item not in self.inventory or their_item not in swapping_vendor.inventory:
+            return False
+        
+        else:
+            self.remove(my_item)
+            self.add(their_item)
+            swapping_vendor.remove(their_item)
+            swapping_vendor.add(my_item)        
```

Comment:
Great use of `add` and `remove` as helper methods! Consider removing this from the `else ` clause
```suggestion
        self.remove(my_item)
        self.add(their_item)
        swapping_vendor.remove(their_item)
        swapping_vendor.add(my_item)        
```